202001.0/5f4778e0-8c57-4959-b3b4-a8c37f477f3b.md

```diff
@@ -24,9 +24,108 @@ APPLICATION_STORE=US console propel:schema:copy
 console propel:install
 console transfer:generate
 ```
+4. Update the code that uses the product availabilities API response to handle the numeric strings. The API response changed in the following way: 
```

Comment:
```suggestion
4. Update apps and integrations that use product availabilities API. From now on it requires the support of high precision availability quantity in string representation (15 will become '15.0000000000'). The API response changed in the following way: 
```

Reply:
The PR has been merged manually, because of the technical issue.